### DISCUSSION TITLE
求助-AdGuardHome  与  openclash 如何设置？

### DISCUSSION THREAD

**Discussion by Jacob-LYC** [AdguardTeam/AdGuardHome#4523] [Q&A]
使用Proxmox VE创建了2个虚拟机，其中1个是openwrt（包含openclash），另1个是AdGuardHome，请问需要如何设置才能让AdGuardHome  和  openclash正常工作呢？

**Comment by KiceHub** [comment]
将AdGuardHome的将上游 DNS 服务器设置为openclash的dns监听端口。
